style.css

```diff
@@ -0,0 +1,586 @@
+  background-repeat: no-repeat;
+  width: 20rem;
+  height: 13.5rem;
+  color: #fff;
+  background-color: var(--clr-accent);
+  background-blend-mode: multiply;
+}
+
+.history-heading {
+  font-size: 2.5rem;
+  margin: 0;
+}
+
+i.fa-solid.fa-chevron-down {
+  color: var(--clr-accent);
+}
+
+.history-text {
+  margin: 0;
+}
+
+.hidden {
+  display: none;
+}
+
+@media (max-width: 767px) {
+  .nav,
+  .nav__logo,
+  .nav-top,
+  .program {
+    display: none;
+  }
+
+  .mobile-menu {
+    list-style: none;
+    width: 100%;
+    height: 100vh;
+    padding: 20px;
+    position: fixed;
+    overflow: hidden;
+    overscroll-behavior: none;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    z-index: 100;
+    color: #fff;
+    background: var(--clr-dark);
+    font-size: var(--fw-bold);
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+  }
+
+  .nav__list {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 10px 40px;
+  }
+
+  .nav__link {
+    text-decoration: none;
+    font-size: 3rem;
+  }
+
+  i.fa-solid.fa-xmark,
+  .nav__link,
+  a.nav__button {
+    color: #fff;
+  }
+
+  .nav-toggle {
+    border: none;
+    background-color: var(--clr-light);
+  }
+
+  i.fa-solid.fa-bars {
+    margin-right: 90%;
+  }
+
+  #menu-open {
+    font-size: 2rem;
+    margin-right: 90%;
+  }
+}
+
+@media (min-width: 768px) {
+  #menu-open,
+  .nav-toggle,
+  .more-info,
+  .intro-greeting.about,
+  .main-sponsors.about,
+  .more {
+    display: none;
+  }
+
+  .page-heading {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 50px 0;
+  }
+
+  .nav__list {
+    display: flex;
+    gap: 10px;
+    padding: 0;
+    list-style: none;
+    cursor: pointer;
+  }
+
+  .nav__item > * {
+    text-decoration: none;
+    color: var(--clr-dark);
+  }
+
+  .nav-top {
+    padding: 0 50px 0;
+    width: 100%;
+    background-color: var(--clr-dark);
+    color: #fff;
+  }
+
+  .social__list {
+    font-size: 1rem;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    list-style: none;
+    gap: 10px;
+  }
+
+  .social__link {
+    text-decoration: none;
+    color: var(--clr-light2);
+  }
+
+  .active {
+    color: var(--clr-accent);
+  }
+
+  a.nav__button {
+    border: 3px solid var(--clr-accent);
+    padding: 0 5px;
+    text-decoration: none;
+    color: var(--clr-accent);
+  }
+
+  .intro-container {
+    width: 70%;
+    margin: 0 auto;
+  }
+
+  .main-details img {
+    margin: 0;
+  }
+
+  .history-image-container {
+    flex-direction: row;
+  }
+
+  .image1,
+  .image2 {
+    width: 20rem;
+  }
+
+  .events-cards {
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+    margin: 30px 0;
+  }
+
+  .events-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 12rem;
+  }
```

Comment:
- Hey, on the desktop version, let's align the text to the center to make it close to the original design.
- let's also add the missing white border on each card.

*design*
![Screenshot from 2023-04-20 00-21-40](https://user-images.githubusercontent.com/59397159/233220391-842b23e3-e893-40fc-832d-493e7c6ec922.png)

*your design*
![Screenshot from 2023-04-20 00-25-35](https://user-images.githubusercontent.com/59397159/233220656-8a4c3cb0-5f9d-438f-b3fd-f769827b5d40.png)